Kong/docs.konghq.com
# refactor(plugins/kuma): refactor custom block plugins specific to Kuma

### Description

Removed separate `tip.rb` and `warning.rb` plugins, which were just copies of the same code with only difference in a block name. Now the code is the same and just registered for different tags.

It also correctly maps different class names between Kong and Kuma docs.


 


This is the followup on https://github.com/kumahq/kuma-website/pull/2012

### Testing instructions

Just go to any documentation page containing a warning or tip block to see if it renders the same as it was before

Preview link: 

### Checklist 

  - not applicable
